lib/hooks/ReturnStatement.js

```diff
@@ -42,7 +42,7 @@ exports.getIndentEdges = function(node, opts) {
   return parentheses ?
     {
       startToken: parentheses.opening,
-      endToken: parentheses.closing
+      endToken: parentheses.closing.prev
```

Comment:
it's partially because of that, the main difference was in the way I was grabbing the beginning of the line on the `indentInBetween` method (it was removing white space tokens, now it doesn't) I also changed the way the `inBetween` starts the loop - always skip `startToken` and `endToken` so it's really *in between* and not from start till end.